### DISCUSSION TITLE
Custom wallet's mnemonics or private key

### DISCUSSION THREAD

**Discussion by canry41341** [trustwallet/wallet-core#2147] [Q&A]
Hello!
I'm wondering if it's possible to create a wallet with our own custom private key or using a mnemonics generated by our own?

**Comment by korinsek** [comment]
Yes you can create wallet and mnemonic out of it
Example:
```
hdWallet = HDWallet(128, "")
val mnemonic = hdWallet.mnemonic()
```
Or create wallet with already existing mnemonics. Example:
```
hdWallet = HDWallet(phrase, "", true)
```
